setup.py

```diff
@@ -55,6 +55,7 @@
     "numpy >=1.15",
     "scipy",
     "h5py",
+    "desktop-app"
```

Comment:
Presumably we won't merge this until desktop-app is on PyPI following 0.1.0 release?

Reply:
Eh, its other requirements aren't all on PyPI either :p.

But I'll just release desktop-app 0.1 on PyPI.

Reply:
Done